src/app/stylesheets/variables.less

```diff
@@ -305,17 +309,19 @@
 /*-----------------------------------------------------------------------*/
 /*                     Responsive Table (cell table)                     */
 /*-----------------------------------------------------------------------*/
-@table-cell-row-spacing:               10px;
+@table-cell-header-row-spacing:        10px;
+@table-cell-header-row-padding:        3px;
+@table-cell-row-spacing:               0px;
 @table-cell-row-padding:               3px;
 
 @table-cell-header-text-size:          12px;
-@table-cell-header-text-color:         @bg-inverse-color;
-@table-cell-header-bg-color:           @bg-main-color;
+@table-cell-header-text-color:         @mno-demo-white;
```

Comment:
why not bg-on-bg-inverse-color ?
I don't think it is a good idea to have the `mno-demo-*` propagate in the variable.less. It is not really frameworky.
I think they should stay in theme.less
I would actually advocate to have also  
`@brand-success @brand-warning @brand-info @brand-danger @brand-primary`
in theme.less.